actions/views/channel.js

```diff
@@ -12,3 +17,10 @@ export function checkAndSetMobileView() {
         });
     };
 }
+
+export function goToLastViewedChannel() {
+    return async (dispatch, getState) => {
+        const lastViewedChannel = getChannelByName(getState(), getLastViewedChannelName(getState()));
+        selectChannel(lastViewedChannel.id)(dispatch, getState);
```

Comment:
Another nit, passing both arguments is unnecessary since you can just call dispatch to do it for you
```suggestion
        dispatch(selectChannel(lastViewedChannel.id));
```